homeassistant/components/ws66i/strings.json

```diff
@@ -0,0 +1,40 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to the device",
+        "data": {
+          "ip_address": "[%key:common::config_flow::data::ip%]",
+          "source_1": "Name of source #1",
```

Comment:
We shouldn't require theses to setup the device. If you can't detect them automatically please move them to an options flow only.  We only want the data needed to connect in the base config flow setup